WordPress/src/main/java/org/wordpress/android/designsystem/DesignSystemColorsScreen.kt

```diff
@@ -0,0 +1,120 @@
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.R
+
+@Composable
+fun DesignSystemColorsScreen(
+    modifier: Modifier = Modifier
+) {
+    LazyColumn(
+        modifier = modifier,
+        horizontalAlignment = Alignment.Start,
+        verticalArrangement = Arrangement.spacedBy(
+            dimensionResource(id = R.dimen.reader_follow_sheet_button_margin_top)
+        )
+    ) {
+        item {
+            ColorTitle("Foreground")
+            val listForeground: List<ColorOption> = listOf(
+                ColorOption("Primary", MaterialTheme.colorScheme.primary),
+                ColorOption("Secondary", MaterialTheme.colorScheme.secondary),
+                ColorOption("Tertiary", MaterialTheme.colorScheme.tertiary),
+                ColorOption("Quartenary", MaterialTheme.colorScheme.quartenary),
+                ColorOption("Brand", MaterialTheme.colorScheme.brand),
+                ColorOption("Error", MaterialTheme.colorScheme.error),
+                ColorOption("Warning", MaterialTheme.colorScheme.warning),
+                ColorOption("WP", MaterialTheme.colorScheme.wp),
+                )
+            ColorCardList(listForeground)
+
+            ColorTitle("Background")
+            val listBackground: List<ColorOption> = listOf(
+                ColorOption("Primary", MaterialTheme.colorScheme.primaryContainer),
+                ColorOption("Secondary", MaterialTheme.colorScheme.secondaryContainer),
+                ColorOption("Tertiary", MaterialTheme.colorScheme.tertiaryContainer),
+                ColorOption("Brand", MaterialTheme.colorScheme.brandContainer),
+                ColorOption("WP", MaterialTheme.colorScheme.wpContainer),
+            )
+            ColorCardList(listBackground)
+        }
+    }
+}
+@OptIn(ExperimentalStdlibApi::class)
+@Composable
+fun ColorCard (colorName: String, color: Color) {
+    Row (modifier = Modifier.padding(10.dp, 3.dp).fillMaxWidth()) {
+        Column {
+            Box(
+                modifier = Modifier
+                    .size(45.dp)
+                    .clip(shape = RoundedCornerShape(5.dp, 5.dp, 5.dp, 5.dp))
+                    .background(color)
+                    .border(width = 1.dp,
+                        color = MaterialTheme.colorScheme.primary,
+                        shape = RoundedCornerShape(5.dp))
+            )
+        }
+        Column {
+            Text(
+                modifier = Modifier.padding(start = 25.dp, end = 40.dp),
+                text = colorName,
+                color = MaterialTheme.colorScheme.primary,
+            )
+            Text(
+                modifier = Modifier.padding(start = 25.dp, end = 40.dp),
+                text = "#" + color.value.toHexString().uppercase().substring(0,8),
+                color = MaterialTheme.colorScheme.secondary
+            )
+        }
+    }
```

Comment:
Add a Divider(modifier = Modifier.padding(start = 10.dp, end = 10.dp)) after this }

Reply:
Thank you, updated

Reply:
Did you forget to push the change?

Reply:
Hey Ravi, thanks for checking - I'm still working on the changes required for the navigation title. I'll tag you / re-request a review when this PR is updated and ready to be looked at again.